README.md

```diff
@@ -0,0 +1,40 @@
+# Android SDK Utils
+
+This is a library that contains utilities which are useful when developing Android SDKs. They are intended for use internally by Rakuten's SDKs.
+
+## Setup
+
+Add to your `build.gradle`:
+
+```groovy
+repositories {
+  jcenter()
+}
+
+dependency {
+  implementation 'com.rakuten.tech.mobile.sdkutils:sdkutils:0.0.1'
+}
+```
+
+## Usage
+
+### OkHttp Header Interceptor
+
+This library contains an extension function for `OkHttpClient.Builder` so you can more easily add headers to all network requests which are made using that client.
+
+```kotlin
+private val client = OkHttpClient.Builder()
+    .addHeaderInterceptor(
```

Comment:
Does this need namespaced with `RAS` in case there is another extension with that name in a different app dependency?

Reply:
Yeah, it's already namespaced to the package of the file it's in. So it can't be used without importing `com.rakuten.tech.mobile.sdkutils.okhttp.addHeaderInterceptor`. Also, we can make the dependency on sdk-utils be an `implementation` dependency when it's added to an SDK which will make it be inaccessible to any Apps that integrate the SDK.